.github/workflows/deploy-feed-discovery-image.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build and publish a feed-discovery image to ghcr.io
+on:
+  release:
+    types: [published]
+
+  push:
+    branches:
+      - master
+
+jobs:
+  docker_publish:
+    runs-on: 'ubuntu-20.04'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # https://github.com/marketplace/actions/push-to-ghcr
+      - name: Build and publish a Docker image for ${{ github.repository }}
+        uses: macbre/push-to-ghcr@master
```

Comment:
I don't think this action is going to work.  They have a bug with uppercase letters in repo names.

Let's use the official Docker one, https://github.com/docker/build-push-action, and we can use https://github.com/docker/login-action to login to ghcr.io